packages/lesswrong/components/recommendations/RecommendationsAndCurated.tsx

```diff
@@ -115,25 +114,17 @@ const RecommendationsAndCurated = ({
   configName: string,
   classes: ClassesType,
 }) => {
+  const {expanded, toggleExpanded} = useExpandedFrontpageSection({
+    section: "recommendations",
+    onExpandEvent: "recommendationsSectionExpanded",
+    onCollapseEvent: "recommendationsSectionCollapsed",
+    defaultExpanded: isEAForum ? "loggedIn" : "all",
```

Comment:
Wait, isn't this the opposite of what we want?  i.e. the "Classic posts" section should be default expanded for logged out users and default collapsed for logged in, right?

Reply:
Good catch - I got confused!